vuepress/docs/next/tutorials/compose/content-tutorial.md

```diff
@@ -11,54 +11,52 @@ This tutorial describes how to create, edit and publish content in an applicatio
-
-         * `View Only Groups`: Add groups to provide viewing access to the content.
+         * `Owner Group`: Choose an available user group to manage the content from the drop-down list. Click the `+` symbol to select. This field is required and can be amended in the App Builder's `User` section under `Groups`.
 
+         * `View Only Groups`: Add groups to provide viewing access to content. 
 
-    *   `Categories`: Use `+` under `Join` to add to the applicable category tree for use in searching and sorting.  This field is not mandatory.
+    *   `Categories`: Use `+` under `Join` to include the `Content` in the appropriate category for searching and sorting.  This field is not mandatory.
 
-    *  `Content Attributes`: Define each required `Attributes` field and any optional fields for the `Content Type` of this item. Only the `Date` and `Title` are required.
+    *  `Content Attributes`: Enter the required `Date` and `Title` fields under the `Attributes` section. You can enter the remaining optional fields as needed.
 
-    * `Content Info - History`: This displays all the saved iterations for the item where you can view details or revert to a previous version. 
+    * `Content Info - History`: This displays the iterations for the saved item. You can view the details or revert to a previous version. 
 
 ![Add Content Information](./img/content-addinfo.png)
 
-## Saving and Approving Content 
+## Save and Approve Content 
 1. For new or edited content, determine which `Status` it belongs to:  
-`Status`:
- * `Draft`: The content is in the development stage and not ready for approval or publication. The `Save` or `Save and Continue` buttons establish this status for all content.
 
- * `Ready`: The content is ready for review and has not yet been approved nor published.  Items with this status can be saved and approved later or published directly.
+ * `Draft`: The `Content` is in the development stage and not ready for approval or publication. The `Save` or `Save and Continue` buttons establish `Draft` status for the content.
+
+ * `Ready`: The `Content` is ready for review, but has not been approved or published.  Items with this status can be saved and approved later or published immediately.
 
 2. Saving options:
 
--   **Save**: The content will be saved in a draft version.
+-   `Save`: The content is saved in a `Draft` version.
 
--   **Save and Continue**: The content will be saved as a draft version and the editable form will be displayed.
+-   `Save and Continue`: The content is saved as a draft version and the editable form is displayed.
 
--   **Save and Approve**: The content will be saved, approved, and ready for use in an application.
+-   `Save and Approve`: The content is saved, approved, and ready for use in an application.
```

Comment:
i think you did not use comma before "and" earlier

Reply:
it's optional so ok either way.